moviepy/video/tools/credits.py

```diff
@@ -10,86 +8,85 @@
 
 
 class CreditsClip(TextClip):
-    @convert_path_to_string("creditfile")
-    def __init__(
-        self,
-        creditfile,
-        width,
-        stretch=30,
-        color="white",
-        stroke_color="black",
-        stroke_width=2,
-        font="Impact-Normal",
-        font_size=60,
-        bg_color=None,
-        gap=0,
-    ):
-        """
+    """Experimental credits clip.
```

Comment:
Why is it "Experimental"?

Reply:
I don't know :sweat_smile: Let me change it